bin/pic-submit

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+#
+# Copyright 2013-2018 Axel Huebl, Rene Widera, Richard Pausch
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+# This script parses template files for a batch systems and fills them
+# with a platform-independent configuration, which can then be submitted
+# to a batch system or plainly executed. It can also execute additional
+# commands at that point, e.g. to copy and archive input files alongside
+# a batch job.
+
+function absolute_path()
+{
+    cd $1
+    pwd
+}
+
+help()
+{
+    echo "pic-submit"
+    echo "submit a job created by tbg"
+    echo ""
+    echo "usage: pic-submit [-s submitcommand] [-h] simulationDirectory"
+    echo ""
+    echo "-s | --submit   command        - Submit command (qsub, \"qsub -h\", sbatch, ...)"
+    echo "                                 Default: submitcommand via export TBG_SUBMIT"
```

Comment:
- missing space between `submit` and `command`
- maybe put `"... via \"export TBG_SUBMIT\""` in quotes

Reply:
`submitcommand` is the submit command from the syntax description above. Maybe I should chnage it to `submit_command`?